tests/kani/Vectors/vector_extend_loop.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// https://github.com/model-checking/kani/issues/184
+
```

Comment:
I would remove the link to avoid confusion (there are no issues with this test anymore).